zephyr/esp32c3/include/sdkconfig.h

```diff
@@ -14,3 +14,4 @@
 #define BTDM_CTRL_HCI_MODE_VHCI 1
 
 #define CONFIG_ESPTOOLPY_FLASHMODE_DIO 1
+#define CONFIG_PARTITION_TABLE_OFFSET 0x8000
```

Comment:
Upstream version hardcodes this to 0x10000 to all SoCs... perhaps to support secure boot properly.  SHould we keep the same approach?
```c
#define CONFIG_PARTITION_TABLE_OFFSET 0x10000
```